src/components/CheckListItem/CheckListItemDetails.tsx

```diff
@@ -22,18 +22,28 @@ const getStyles = (theme: GrafanaTheme) => ({
 interface Props {
   frequency: number;
   activeSeries?: number;
+  probeLocations?: number;
```

Comment:
I don't think this value is possibly undefined. If `check.probes`  is undefined when it's passed as a prop to this component in `CheckListItem`, calling `.length` on it will cause the page to explode. So we probably need to make `probeLocations` behave more like the `frequency` prop instead of like the `activeSeries` prop.

Reply:
This was a really long winded way of saying I don't think you need the `?` and `undefined` checks in this component

Reply:
updated